webserver/webserver/src/main/java/io/helidon/webserver/WebTracingConfig.java

```diff
@@ -141,6 +141,35 @@ Service service() {
         private final List<PathTracingConfig> pathTracingConfigs = new LinkedList<>();
         private TracingConfig tracedConfig = TracingConfig.ENABLED;
 
+        /**
+         * OpenTracing spec states that certain MP paths need to be disabled by default.
+         * Note that if a user changes the default location of any of these using
+         * web-context's, then they would need to provide these exclusions manually.
+         *
+         * The default path configs below are overridable via configuration. For example,
+         * health could be enabled by setting {@code tracing.paths.0.path=/health} and
+         * {@code tracing.paths.0.enabled=true}.
+         */
+        Builder() {
+            addPathConfig(PathTracingConfig.builder()
+                    .path("/metrics")
+                    .tracingConfig(TracingConfig.DISABLED)
+                    .build());
+            // Simplified matching for base, vendor and application paths
+            addPathConfig(PathTracingConfig.builder()
+                    .path("/metrics/{+}")
+                    .tracingConfig(TracingConfig.DISABLED)
+                    .build());
+            addPathConfig(PathTracingConfig.builder()
+                    .path("/health")
```

Comment:
Even if the tracing spec does not mention them, there are subpaths for `/health` as well as for `/metrics`: `/health/ready`, `/health/live`, and (in the latest spec but not the one which applies to Helidon 2.x) `/health/started`. (https://download.eclipse.org/microprofile/microprofile-health-4.0/microprofile-health-spec-4.0.html#_appendix_a_rest_interfaces_specifications)

These should probably be excluded by default as well.

Reply:
Good point, I forgot about those. Will exclude them as well.